### ISSUE TITLE
cannot find symbol getLivenessDetected()

### ISSUE THREAD

**Issue by johnreitano** [directpayintl/cordova-plugin-jumio-netverify#1]
Hi,

I came across your plugin and got this compilation error:

```
Error: /Users/John/dev/ur_money/platforms/android/gradlew: Command failed with exit code 1 Error output:
/Users/John/dev/ur_money/platforms/android/src/com/cordova/plugin/android/jumiomobilecordovaPlugin/JumioMobileCordovaPlugin.java:112: error: cannot find symbol
                    documentDataResult.put("livenessDetected", documentData.getLivenessDetected());
                                                                           ^
  symbol:   method getLivenessDetected()
  location: variable documentData of type NetverifyDocumentData
```

I commented out line 112 and was able to get the plugin built.

On a possibly related note, It looks like you are using netverify v 2.2.0 which Jumio is no longer supporting. Do you have any plans to upgrade to v 2.4.0?

Thanks,
John Reitano

PS. If you are available to do a freelance project getting this plugin to work in our Android and iOS app, please contact me at jreitano@ur.technology

**Comment by lhoezee** [comment]
Hello, we have not started v 2.4 yet because there are so many changes. Our current implementation is using SDK 2.3 which seems to be working well with us for now.  Regarding commenting out that link, that's correct. I actually had this same problem locally yesterday. We'll be updating it this week to include the SDK 2.3 updates as well.

**Comment by johnreitano** [comment]
Hi lhoezee, Thanks for the response. Looking forward to seeing your next version. 

I'm now using your code but modified the SDK dependency to use  2.3, and I now get a lot further:  I'm able to scan and upload the id document. But a few seconds after that the app crashes before the callback passed to presentNetverifyController is called. Have you seen this behavior?

**Comment by lhoezee** [comment]
Yes, we had the same problem. @kkrebs7 was able to resolve that issue. So I'm thinking our new release will have this fixed as well.

**Comment by johnreitano** [comment]
Looking forward to seeing it. Also, I sent you a private message. Please let  me know if you did not receive it.

**Comment by lhoezee** [comment]
@johnreitano I did not receive a private message.

**Comment by johnreitano** [comment]
Ok, I sent it via the form on http://www.thecodify.com/ . Can you send me an email to jreitano@ur.technology and I'll send you an email?

**Comment by johnreitano** [comment]
Just sent a message via LinkedIn.
